survey_legal/static/src/js/survey_form.esm.js

```diff
@@ -2,18 +2,17 @@
 /* Copyright 2022 Tecnativa - David Vidal
    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 */
-import SurveyFormWidget from "survey.form";
-import {_t} from "web.core";
+import SurveyFormWidget from "@survey/js/survey_form";
+import {_t} from "@web/core/l10n/translation";
 
 SurveyFormWidget.include({
     /**
      * Validate legal terms acceptance if present
      *
      * @override
-     * @param {JQuery} [$form] the survey form
-     * @returns {Boolean}
-     */
-    _validateForm: function ($form) {
+     * */
+    /* eslint-disable-next-line no-unused-vars */
+    _validateForm: function ($form, formData) {
```

Comment:
```suggestion
    _validateForm: function ($form) {
```
In js you don't need to set all the params to extend a function 😄

Reply:
thanks